client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -40,7 +40,8 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 				{ task.displayBadge && task.badgeText ? (
 					<Badge type="info-blue">{ task.badgeText }</Badge>
 				) : null }
-				{ ! taskDisabled && (
+				{ /* don't display chevron and badge at the same time */ }
+				{ ! taskDisabled && ! task.displayBadge && ! task.badgeText && (
```

Comment:
Is checking `! taskDisabled && ! task.displayBadge ` sufficient here?
Do we need to check the task.badgeText value?

Reply:
@agrullon95 That's a good question, I included the `task.badgeText` check for the sake of consistency with `line 40`. I am assuming there's no way `task.displayBadge` will be true but `task.badgeText` will be empty. If that is the case I can remove the `task.badgeText` check